README.md

```diff
@@ -22,8 +29,54 @@ require('@newrelic/mysql')
 - [`mysql`](https://www.npmjs.com/package/mysql)
 - [`mysql2`](https://www.npmjs.com/package/mysql2)
 
-For more information, please see the agent [installation guide][1], and
-[compatibility and requirements][2].
+For more information, please see the agent [installation guide][3] and the agent [compatibility and
+requirements][4].
+
+Our [API and developer documentation](http://newrelic.github.io/node-newrelic/docs/) for writing
+instrumentation will be of help. We particularly recommend the tutorials and various "shim" API
+documentation.
+
+## Testing
+
+The module includes a suite of unit and functional tests which should be used to verify that your
+changes don't break existing functionality.
+
+All tests are stored in `tests/` and are written using [Tap](https://www.npmjs.com/package/tap) with
+the extension `.tap.js`.
+
+To run the full suite, run: `npm test`.
+
+Individual test scripts include:
+
+```
+npm run unit
+npm run versioned
+```
+
```

Comment:
You may want to address how the versioned tests can't run without setting up the mysql docker container from teh main agent.
https://github.com/newrelic/node-newrelic-mysql/issues/25#issuecomment-668878636
Plus, I don't think there are any unit tests that run.  Might want to verify and remove the unit test script from package.json.

Reply:
Done and done